pkg/api/validation/validation.go

```diff
@@ -52,6 +52,7 @@ const fieldImmutableErrorMsg string = `field is immutable`
 const isNotIntegerErrorMsg string = `must be an integer`
 
 var pdPartitionErrorMsg string = validation.InclusiveRangeError(1, 255)
+var volumeModeErrorMsg string = validation.InclusiveRangeError(0, 0777)
```

Comment:
it will show a number, but it doesn't know to print it as 3-digit-octal, so it will come out ugly and meaningless to end users.